hvillacrusisCT/dotTheory---Javascript
# JS-686: Write Javascript Statements

## Related Ticket
[JS-686](https://codeandtheory.atlassian.net/browse/JS-686)

## Description
Added the Write JavaScript Statements challenge solutions.

## Testing
1. Open index.html
2. "Begin Program" should be displayed in the console
3. Alert dialog should pop up displaying "I am Programming!"
4. Click Ok button from the Alert dialog
5. "End Program" should be displayed in the console
